fs/sync.c

```diff
@@ -18,7 +18,7 @@
 #include <linux/backing-dev.h>
 #include "internal.h"
 
-bool fsync_enabled = true;
+bool fsync_enabled = false;
```

Comment:
We need to make sure that memory chips on all supported devices don't corrupt data with fsync disabled